domain/application/charm/store/store.go

```diff
@@ -247,7 +247,7 @@ func (c *charmReaderCloser) Close() error {
 		// We don't need to log this as error, as the file will be removed
 		// when the process exits or by the OS. It's not a direct action
 		// by the user, hence Info.
-		c.logger.Infof(context.TODO(), "removing temporary file: %v", removeErr)
+		c.logger.Infof(context.Background(), "removing temporary file: %v", removeErr)
```

Comment:
Should ones like this not stay as TODO?

Reply:
No. We don't always need a `context.Context` passed into the function/method. If we're happy that the log holds no real value from a tracing perspective, or the function/method doesn't call additional methods, the `context.Background()` is a witness to that effect. There is nothing more we need to do with this one, move along :wink: